osm2gtfs.py

```diff
@@ -31,18 +27,32 @@
 
 
 def main():
+
+    # Load config json file
+    try:
+        config = json.load(args.config)
```

Comment:
Can we handle this in a separate issue, please? I think as it is proposed here, it's already an improvement.